.gitignore

```diff
@@ -77,3 +77,8 @@ spark-warehouse/
 # For R session data
 .RData
 .RHistory
+.Rhistory
```

Comment:
Duplicate line. Can these be pushed down into a `.gitignore` in the subdirectory? this file is a mess.